template.json

```diff
@@ -13,8 +13,9 @@
     "reactstrap": "7.1.0",
     "redux": "4.0.1",
     "redux-thunk": "2.3.0",
-    "kpmp-common-styles":"^1.0.4",
-    "kpmp-common-components": "^1.1.2"
+    "kpmp-common-styles":"^1.0.6",
+    "kpmp-common-components": "^1.1.2",
+    "react-table-filter": "^2.0.2"
```

Comment:
Do we want to include this in the app template?

Reply:
Same as above, needed something for a table, this would be swappable with whatever we end up going with